_posts/2017-08-28-upgrade-to-rails-3.markdown

```diff
@@ -147,6 +148,24 @@ end
 ```
 - Mailers should now be in app/mailers instead of app/models.
 
+<h3 id="erb-syntax">ERB Syntax</h3>
+ Block helpers that use concat (e.g., form_for, form_tag) will need to replace `<%` with `<%=`. The current syntax will continue to work for now, but you will get deprecation warnings since it will go away in the future. 
+ 
+<h3 id="ajax-helpers">AJAX Helpers</h3>
+AJAX JavaScript helpers have moved to be unobtrusive and use `:remote => true`. Depending on the helper call some unobtrusive JavaScript may need to be added. For example:
+```
+link_to_remote ("Update Example", 
+                        :update => 'example',
+                        :url => {:action => 'example'})
+```
+Will need to change to:
+```
+link_to "Update Example", { :action => 'example' }, remote => true 
+```
+and unobtrusive JavaScript will need to be added for the what "update" is doing.
```

Comment:
```suggestion
and a JavaScript event listener will need to be added for what "update" is doing.
```

Maybe we can add a small code snippet explaining this line? no too fancy, just the `$(...).on('ajax:success', function(response) {.... $('#example').html(response.responseText)}` or something like that. The link below is not working for me (maybe this is shown there but I think it's better to have an example on our article)